Eevee.Sleep.Bot/appsettings.json

```diff
@@ -30,7 +30,10 @@
     "Token": "<TOKEN>",
     "Guild": "<GUILD_ID>",
     "Channels": {
-      "AdminAlert": 0
+      "AdminAlert": 0,
+      "JPInGameAnnouncementNotice": 0,
+      "ENInGameAnnouncementNotice": 0,
+      "ZHInGameAnnouncementNotice": 0
```

Comment:
> Can be any way that once announcement content is found to be updated, sends out the notification to multiple designated channels to be specified in the config json file.

Multiple channel designation may not be what this is about... 🤔

Reply:
This is exactly what we intended to do. Make this like below instead though (basically nested):
```
"InGameAnnouncementNotification": {
  "JP": 0,
  "EN": 0,
  "ZH": 0,
}
```